src/libs/actions/UpdateRequired.ts

```diff
@@ -0,0 +1,20 @@
+import Config from 'react-native-config';
+import Onyx from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function alertUser() {
+    // For now, we will pretty much never have to do this on a platform other than production.
+    // We should only update the minimum app version in the API after all platforms of a new version have been deployed to PRODUCTION.
+    // As staging is always ahead of production there is no reason to "force update" those apps.
+    if ((Config?.ENVIRONMENT ?? CONST.ENVIRONMENT.DEV) !== CONST.ENVIRONMENT.PRODUCTION) {
```

Comment:
Any specific reasons to default to DEV here?

Reply:
Not really! I think I copied this from somewhere. It could be an empty string. Just want to make suer we are on production and not confusing people on staging or dev.